db-commons/src/main/resources/db.conf

```diff
@@ -51,4 +51,13 @@ bitcoin-s {
       url = "jdbc:sqlite:"${bitcoin-s.chain.db.path}${bitcoin-s.chain.db.name}
     }
   }
+
+  test = ${common}
```

Comment:
This probably should not be in `src/main`, the configuration should be in `test`